lib/ClangImporter/ClangClassTemplateNamePrinter.cpp

```diff
@@ -179,10 +179,11 @@ struct TemplateArgumentPrinter
   void VisitTypeTemplateArgument(const clang::TemplateArgument &arg,
                                  llvm::raw_svector_ostream &buffer) {
     auto ty = arg.getAsType();
+    if (ty.isConstQualified())
+      buffer << "__cxxConst<";
```

Comment:
Actually, Egor just mentioned that we have other qualifiers in C++, like `volatile`. Maybe we should handle those as well.